src/cookbooks/operational_monitoring.md

```diff
@@ -0,0 +1,120 @@
+# Introduction to Operational Monitoring
+
+Operational monitoring is a general term that refers to monitoring the health of software.
+
+However, Mozilla's internal Operational Monitoring tooling supports a couple of specific use cases:
+
+1. Monitoring build over build. This is typically for Nightly where one build may contain changes that a previous build doesn't and we want to see if those changes affected certain metrics.
+2. Monitoring by submission date over time. This is helpful for a rollout in Release for example, where we want to make sure there are no performance or stability regressions over time as a new build rolls out.
+
+The monitoring dashboards produced for these use cases are available in [Looker](https://looker.com/).
+
+Access to the Looker Operational Monitoring dashboards is currently limited to Mozilla employees and designated contributors. For more information, see [gaining access](../concepts/gaining_access.md).
+
+## How to use Operational Monitoring
+
+An operational monitoring project is defined using a project definition JSON file in combination with 2 other JSON files.
+
+In order to create a new project to be monitored:
+
+1. A new `<project_name>.json` file must be added to the `operational_monitoring` GCS bucket at `gs://operational_monitoring/projects`.
```

Comment:
I remember having trouble finding the right project that has the bucket at some point. Might be worth adding what GCP project the bucket is in